test/integration/dashboard/student_can_see_events_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class User::NewEventsTest < ActionDispatch::IntegrationTest
+  test 'a student can see a single event that is displayed' do 
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.